File: extrafiles/activity.php
<?php
	$con=mysqli_connect("localhost","mrahman_user","p@ssw0rd","mrahman_data");
	
	$response["activity"] = array();
	if (mysqli_connect_errno($con))
	{
	echo "Failed to connect to MySQL: " . mysqli_connect_error();
	}
	

	$module_id = $_GET['module_id'];
	
	$check_name = mysqli_query($con,"SELECT * FROM activity where module_id = '$module_id'");
	
	$response["success"] = false;

	
	while($row = mysqli_fetch_array($check_name)){
		$product = array();
		$product["activity_number"] = $row["activity_number"];
		$product["description"] = $row["description"];
		$product["activity_objective"] = $row["activity_objective"];
		$product["result"] = $row["result"];
		$product["activity_time"] = $row["activity_time"];
		$product["materials"] = $row["materials"];
		$product["instruction"] = $row["inst"];
		// push single product into final response array
		array_push($response["activity"], $product);
		$response["success"] = true;
	}
	mysqli_close($con);
	echo json_encode($response);
?>

File: app/src/main/java/net/codeforgood/sciencebehindsports/Object/Activity.java
package net.codeforgood.sciencebehindsports.Object;

/**
 * Created by VAIO-Jun on 10/28/2016.
 */

public class Activity {
    private String name;
    private int id;
    private int activity_number;
    private String description;
    private int module_id;
    private String activity_objective;
    private String result;
    private int activity_time;
    private String materials;
    private String instruction;

    public Activity(String name, int id, int activity_number, String description,
                    int module_id, String activity_objective, String result,
                    int activity_time, String materials, String instruction) {
        this.name = name;
        this.id = id;
        this.activity_number = activity_number;
        this.description = description;
        this.module_id = module_id;
        this.activity_objective = activity_objective;
        this.result = result;
        this.activity_time = activity_time;
        this.materials = materials;
        this.instruction = instruction;
    }

    public String getName() {
        return name;
    }

    public int getActivity_number() {
        return activity_number;
    }

    public int getId() {
        return id;
    }

    public int getModule_id() {
        return module_id;
    }

    public int getActivity_time() {
        return activity_time;
    }

    
}
